src/app/modules/main/chat_section/module.nim

```diff
@@ -411,8 +411,8 @@ method activeItemSet*(self: Module, itemId: string) =
     return
 
   # update view maintained by this module
-  self.view.chatsModel().setActiveItem(itemId)
   self.view.activeItemSet(chat_item)
+  self.view.chatsModel().setActiveItem(itemId)
```

Comment:
This fixes the problem where the active item is undefined when switching to a section. Now the active item will be set in the view before switching.